scripts/browser-tests.groovy

```diff
@@ -13,8 +13,12 @@ timeout(60) {
                       # Disable starting Selenium, Docker is used instead
                       sed -i 's/"start_process" : true,/"start_process" : false,/g' nightwatch.json
                       npm install
-                      npm test
                     """
+                    try {
+                        sh "npm test"
+                    } catch (exp) {
+                        currentBuild.result = 'UNSTABLE'
```

Comment:
If you do just this, the exception stack trace and message is not shown in the build log. I think it is pretty important to see the full stack trace in case of error.

I tried using `exp.printStackTrace()` as the very first command in `catch` block but it didn't work. Not sure how to overcome this. But seeing the stack trace seems more important to me than 'FAILURE' vs 'UNSTABLE' result.

Reply:
Ahh OK. Hm so maybe `println(exp.getStackTrace())`. Can give it a try.

Reply:
Nope, I can see only `[Ljava.lang.StackTraceElement;@2404307f` in the log